html_css_exercise/femc_level1_huddleLandingPage/css/index.css

```diff
@@ -0,0 +1,144 @@
+:root {
+    --background-color-1: #674bb1;
+    font-size: 62.5%;
+}
+
+html,
+body {
+    height: 100%;
+    width: 100%;
+    margin: 0;
+    overflow: hidden;
+}
+
+body {
+    background-image: url(../img/bg-desktop.svg);
+    background-color: var(--background-color-1);
+    background-repeat: no-repeat;
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+}
+
+a {
+    cursor: pointer;
+}
+
+/****** Header Styling *********/
+
+.header-main {
+    width: 100%;
+    height: 65px;
+    margin: 0 0 40px;
+    padding: 0 40px;
+}
+
+/****** Main Section Styling *********/
+
+.container-main {
+    width: 100%;
+    padding: 0 40px;
+    display: flex;
+}
+
+.main-img-mockup {
+    flex-grow: 1;
+    max-width: 1000px;
```

Comment:
Instead of flex-grow and max-width, I'd recommend using `flex-basis: 50%` both here and on `.main-message`. This constrains each flex item to 50% of the flex container. This scales better through all the possible screen sizes as you approach mobile size.

Reply:
went with a 45%/ 55%. I like the flex-basis solution. was not happy with flex grow with max width, it worked but I imagine some screen sizes out there would have had issues and this feels more maintainable